### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gauntface**
Vitess is no longer used by YouTube and this change updates the README accordingly

### CODE REVIEW

**Comment by frouioui** [comment]
Hello @gauntface, can you sign commit f7f5693? It is required for the [DCO check](https://github.com/vitessio/vitess/pull/9760/checks?check_run_id=5292701702).

```
git rebase HEAD~1 --signoff
git push --force-with-lease origin patch-1
```

**Review by deepthi — APPROVED** [review]
Thank you @gauntface! This LGTM.
We do need a sign-off (DCO) before we can merge.

**Comment by deepthi** [comment]
@gauntface I assume you would also want to remove the YT logo from the Vitess website.
It will need to be deleted from here https://github.com/vitessio/website/tree/prod/static/img/logos/users
Either you can do a PR for it or one of us (maintainers) can do one.

**Pull request closed without merging**
